docs/updates/2019-04.md

```diff
@@ -0,0 +1,180 @@
+# April 2019
+
+Welcome to the April 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Enhancements in catwalk
+
+### Demo app
+
+Looking at the visitor statistics we could see that a lot of people opened and left catwalk without opening a Qlik
+application. One possible explanation to the behavior could be a curiosity around the tool and its capabilities but
+lacking a running engine or a configured Qlik Sense Enterprise prevented further exploring. To mitigate this we have
+have added a
+[demo app](https://catwalk.core.qlik.com/?engine_url=wss://apps.core.qlik.com/app/doc/01775889-c700-413f-9b0e-6ba1837c52b0/)
+which can be used as an introduction to the tool and is automatically loaded if no engine URL is specified.
+
+### Default web socket
+
+If you are mainly working with one engine instance, using the new default socket connection feature might be of use.
+By clicking on the star, top right in catwalk, the current web socket will be stored in the browser's local storage and
+used in coming catwalk sessions.
+
+![screenshot](../images/catwalk-default-websocket.png)
+
+### Add support for selections in "always one selected value" fields
+
+A field with the [property](https://core.qlik.com/services/qix-engine/apis/qix/definitions/#nxfieldproperties)
+`qOneAndOnlyOne` will enforce only one selected value. The selection model in catwalk has been updated accordingly.
+
+![screenshot](../images/catwalk-field-select-one-value.gif)
+
+### Visual support for loosely coupled tables
+
+We have added support for loosely couple tables, i.e. tables with a circular reference. Normally this is not intended in
+a data model and catwalk will help out by marking the involved fields with a dashed red line.
+
+![screenshot](../images/catwalk-circular-reference.png)
+
+### Export hypercube data and definition
+
+In one of the incoming feature requests there was an expressed need for exporting hypercube data. It sounded like an
+interesting features so we decided to build support for it but also offer a way to export the hypercube definition to
+Clipboard. Both features can be found in the new icons top right in open hypercubes.
+
+![screenshot](../images/catwalk-export-hypercube.png)
+
+### Save open hypercubes
+
+Something which has irritated us for a while is that all hypercubes created in catwalk gets lost when the window is
+refreshed or closed. To prevent this we now store the hypercube definition in the browser's local storage.
+
+### Keep table headers during scrolling
+
+Previously, when you navigate in large applications, the table headers were lost which caused some confusions on what
```

Comment:
Typo 'confusions', perhaps rephrase to something like "which could cause the user to lose track of which table a field belonged to"?